internal/runbits/dependencies/summary.go

```diff
@@ -44,10 +41,9 @@ func OutputSummary(out output.Outputer, directDependencies buildplan.Artifacts)
 			subdepLocale = locale.Tl("summary_subdeps", "([ACTIONABLE]{{.V0}}[/RESET] sub-dependencies)", strconv.Itoa(numSubs))
 		}
 
-		item := fmt.Sprintf("[ACTIONABLE]%s@%s[/RESET] %s", ingredient.Name, ingredient.Version, subdepLocale)
-
-		out.Notice(fmt.Sprintf("[DISABLED]%s[/RESET] %s", prefix, item))
```

Comment:
Why drop the DISABLED item here?

Reply:
There's a new `BulletTreeDisabled` preset I'm using now.